app/views/admin/settings/forms/_campaign.html.erb

```diff
@@ -11,6 +11,15 @@
                } %>
     <div id="campaignBodyContainer" class="card-body collapse hide" aria-labelledby="campaignBodyContainer">
       <fieldset class="grid gap-4">
+        <div class="crayons-field">
+          <%= admin_config_label :display_name, "Display Name" %>
+          <%= admin_config_description Constants::Settings::Campaign::DETAILS[:display_name][:description] %>
+          <%= f.text_field :display_name,
+                           class: "crayons-textfield",
+                           value: Settings::Campaign.name,
```

Comment:
this should be `Settings::Campaign.display_name`

![Screenshot from 2022-02-21 13-54-43](https://user-images.githubusercontent.com/1237369/155019034-3feff1d2-6679-48fa-a59e-59b795a4eacd.png)

Reply:
✅